CRM/Event/Form/SelfSvcUpdate.php

```diff
@@ -149,19 +166,13 @@ public function preProcess() {
     }
     $details = array_merge($details, $selfServiceDetails);
     $this->assign('details', $details);
-    $this->selfsvcupdateUrl = CRM_Utils_System::url('civicrm/event/selfsvcupdate', "reset=1&id={$this->_participant_id}&id=0");
-    $this->selfsvcupdateText = ts('Update');
-    $this->selfsvcupdateButtonText = ts('Update');
-    // Based on those ids retrieve event and verify it is eligible
-    // for self update (event.start_date > today, event can be 'self_updated'
-    // retrieve contact name and email, and let user verify his/her identity
```

Comment:
I didn't find this comment useful (or particarly relevant to the surrounding code) so I removed it.

Reply:
The comment probably goes with the call CRM_Event_BAO_Participant::getSelfServiceEligibility and got moved around, but yeah no need for it to be in this func.